lib/PluginsManager.js

```diff
@@ -41,16 +43,31 @@ PluginsManager.prototype.installTestsForExistingPlugins = function () {
     installedPlugins.forEach(function(plugin) {
         // there is test plugin available
         if (fs.existsSync(path.join(plugin.dir, 'tests', 'plugin.xml'))) {
-            me.installSinglePlugin(path.join(plugin.dir, 'tests'));
+            var additionalArgs = '';
+            if (plugin.id.indexOf('cordova-plugin-file-transfer') >= 0) {
+                var fileServerUrl = Server.prototype.getConnectionAddress.bind({
```

Comment:
Can we just create a new `Server()` and call `getConnectionAddress` on the instance instead? IMO this seems to be more intuitive than using `bind`